api/resolvers/index.js

```diff
@@ -1201,41 +1204,185 @@ const resolvers = {
 
       return kcAdminClient.users.findOne({ id: kauth.sub });
     },
-    // inviteMembers: async (
-    //   parent,
-    //   { emails },
-    //   { currentMember, models: { Member, Event } }
-    // ) => {
-    //   if (!currentMember || !currentMember.isAdmin)
-    //     throw new Error('You need to be admin to invite new members');
+    inviteEventMembers: async (
+      parent,
+      { emails: emailsString, eventId },
+      {
+        currentOrgMember,
+        currentOrg,
+        kcAdminClient,
+        models: { OrgMember, EventMember, Event },
+      }
+    ) => {
+      const currentEventMember = await EventMember.findOne({
+        orgMemberId: currentOrgMember.id,
+        eventId,
+      });
 
-    //   const emailArray = emails.split(',');
+      const event = await Event.findOne({ _id: eventId });
 
-    //   if (emailArray.length > 1000)
-    //     throw new Error('You can only invite 1000 people at a time.');
+      if (!currentEventMember || !currentEventMember.isAdmin)
+        throw new Error("You need to be admin to invite new members");
 
-    //   const memberObjs = emailArray.map((email) => ({
-    //     email: email.trim(),
-    //     eventId: currentMember.eventId,
-    //     isApproved: true,
-    //   }));
+      const emails = emailsString.split(",");
 
-    //   let members = [];
+      if (emails.length > 1000)
+        throw new Error("You can only invite 1000 people at a time");
```

Comment:
Our current limit on mailgun is 1250 per month :p we could increase that of course but maybe also healthy/to avoid spam, capping it at 100 for now could be good?

Reply:
I think Kiezburn had around 600 people to add, so would be nice to allow them to do it once I guess